doc/build-helpers/images/dockertools.section.md

```diff
@@ -373,7 +373,7 @@ See [](#ex-dockerTools-buildLayeredImage-hello) to see how to do that.
 # Building a layered Docker image
 
 The following package builds a layered Docker image that runs the `hello` executable from the `hello` package.
-The Docker image will have name `hello` and tag `latest`.
+The Docker image will have name `hello` and tag `latest`. The created image can have a current timestamp when setting the `created` parameter to "now" - this will only affect the image, not the individual layers.
```

Comment:
This is not the place to add this information. The `buildLayeredImage` section is a thin wrapper for the `streamLayeredImage` section, which is where you should add the extra sentence instead.

Adding this information either in the general area for `streamLayeredImage` (the content before the "Inputs" subheading) or in the documentation of the `created` attribute seems fine to me.

Also, when moving the text, please make sure you put one sentence per line to make it easier to review documentation. See https://github.com/NixOS/nixpkgs/blob/master/doc/README.md#documentation-conventions